### PR TITLE
add income view

### PR DESCRIPTION
**Pull request by VitalyKadyrko**

### CODE REVIEW

File: .idea/finance_tracker.iml [hunk 1]
```diff
@@ -14,113 +14,115 @@
   </component>
```
Comment (Shkuryn) [inline .idea/finance_tracker.iml:14]: For what this file? may be better remove it?

File: db/migrate/20220511142620_add_id_to_income.rb [hunk 2]
```diff
@@ -1,9 +0,0 @@
-# frozen_string_literal: true
```
Comment (Shkuryn) [inline db/migrate/20220511142620_add_id_to_income.rb:1]: Is it correct to delete migration? Possible we should create new?

Comment (VitalyKadyrko) [reply db/migrate/20220511142620_add_id_to_income.rb:1]: No, a migration cannot be deleted. Need to create a new migration that removes the columns which need.

File: app/assets/config/manifest.js [hunk 3]
```diff
@@ -1,2 +1,2 @@
-//= link_tree ../images
```
Comment (Shkuryn) [inline app/assets/config/manifest.js:1]: let's everybody create a folder locally

**Comment by Ikl0** [comment]
reapprove pull request

**Review by Shkuryn — APPROVED** [review]

**Pull request merged**